### PR TITLE
Use videoInput for webcam capture on Windows

### LINKED ISSUE #142: Webcams don't work on Windows
Also opening a camera that does not exist causes an exception.

edit: Opening cameras on windows does not work, if the camera exists on that slot or not. The problem seems to be in the way you handle the camera class with opencv. I have used a program written with java to process an image.

## System Info:

Property Name | Property 
 ----- | -----
javafx.version | 8.0.66
java.runtime.name | Java(TM) SE Runtime Environment
java.vm.version | 25.66-b17
java.vm.vendor | Oracle Corporation
java.vm.name | Java HotSpot(TM) 64-Bit Server VM
java.runtime.version | 1.8.0_66-b17
java.awt.graphicsenv | sun.awt.Win32GraphicsEnvironment
javafx.runtime.version | 8.0.66-b17
os.name | Windows 10
os.version | 10.0
os.arch | amd64
file.encoding | Cp1252
java.vm.info | mixed mode
java.version | 1.8.0_66
sun.arch.data.model | 64
sun.cpu.endian | little

### Exception: cvCreateCameraCapture() Error: Could not create camera capture.
```java
java.io.IOException: A problem occurred trying to start the frame grabber for Webcam 1
	at edu.wpi.grip.core.sources.CameraSource.startVideo(CameraSource.java:97)
	at edu.wpi.grip.core.sources.CameraSource.<init>(CameraSource.java:68)
	at edu.wpi.grip.core.sources.CameraSource.<init>(CameraSource.java:43)
	at edu.wpi.grip.ui.pipeline.AddSourceView.lambda$null$39(AddSourceView.java:94)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.bytedeco.javacv.FrameGrabber$Exception: cvCreateCameraCapture() Error: Could not create camera capture.
	at org.bytedeco.javacv.OpenCVFrameGrabber.start(OpenCVFrameGrabber.java:179)
	at edu.wpi.grip.core.sources.CameraSource.startVideo(CameraSource.java:95)
	... 6 more

```

### PR DESCRIPTION
**Pull request by ThomasJClark**
OpenCV's frame grabber framework seems to only work reliably for us on linux and osx, leading to crashes when using webcams on Windows.

Before this is merged, I'd like to get some testing done on a couple different computers and webcams.

closes #142

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -43,7 +45,8 @@ dependencies {
     compile group: 'com.thoughtworks.xstream', name: 'xstream', version: '1.4.8'
     compile group: 'org.bytedeco', name: 'javacv', version: '1.1'
     compile group: 'org.bytedeco.javacpp-presets', name: 'opencv', version: '3.0.0-1.1'
-    compile group: 'org.bytedeco.javacpp-presets', name: 'opencv', version: '3.0.0-1.1', classifier: osdetector.classifier.replace("osx", "macosx")
+    compile group: 'org.bytedeco.javacpp-presets', name: 'opencv', version: '3.0.0-1.1', classifier: os
+    compile group: 'org.bytedeco.javacpp-presets', name: 'videoinput', version: '0.200-1.1', classifier: os
```
Comment (JLLeitschuh) [inline build.gradle]: Does this even need to be here for any os other than Windows?

Comment (ThomasJClark) [inline build.gradle]: the maven artifact is set up to only add  anything on Windows automatically

Comment (JLLeitschuh) [inline build.gradle]: Okay

**Comment by PaulaRudy** [comment]
Tom, can you update this so it can be merged? Thanks!

**Comment by ThomasJClark** [comment]
done

**Pull request merged**